cfgov/apache/conf.d/redirects.conf

```diff
@@ -1001,14 +488,9 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
 RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/creativesavingsstrategies/?$ /consumer-tools/military-financial-lifecycle/
 RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/protecting/?$ /consumer-tools/military-financial-lifecycle/
 
-# 9/30/20 IA work, practitioner-resources line moved here to allow the above redirects to maintain effect
-Redirect permanent /practitioner-resources/ /consumer-tools/educator-tools/
-
 # Common misspellings
 RedirectMatch (?i)(.*)morgage(.*) $1mortgage$2
 
-Redirect /assets https://files.consumerfinance.gov/a/assets
```

Comment:
Same as the `/static/askcfpb/` comment above: it feels weird to have a redirect meant for static assets living in the CMS, but it seems to work, so probably not a reason to hold up this PR.

Reply:
Agreed!